### ISSUE TITLE
Stars texture jitter

### ISSUE THREAD

**Issue by NuclearPhoenixx** [krzmig/godot-simple-sky-project#6]
Hey there! Really awesome work I absolutely love the shader you made.

One thing I don't fully understand is if it's intended what's going on with the stars texture if I add it to the shader.
Whenever I don't move the camera, so the image is static, it looks pretty neat. However, as soon as I move it, things start to jitter around.
They're only fully visible if the camera is static.

Here is an example video:

https://github.com/krzmig/godot-simple-sky-project/assets/17517687/8e867729-5381-42f8-8449-c803b926c131

Maybe someone on here has an idea if that's deliberate or how to fix it if not. 🙂

**Comment by tojutaha** [comment]
This is really late answer, but this happens when you use TAA. To fix it, dont use TAA. Its really bad. :)

**Comment by NuclearPhoenixx** [comment]
Yes, thanks that did it! That's really a shame though as TAA and FXAA are the only AA options in my game, because MSAA does not run well with the module I'm using :/

Is there any way around this? Why is that, do you know this by chance?

**Comment by NuclearPhoenixx** [comment]
Nevermind, I know why now. TAA is stupid, not an issue with the project 😆
